run_app.sh

```diff
@@ -2,16 +2,16 @@
 
 set -e
 
-web_server=${WEB_SERVER:-gunicorn}
-
-if [ "$web_server" = "gunicorn" ]; then
-    run_command="gunicorn application:application"
-elif [ "$web_server" = "uwsgi" ]; then
+if [ "$WEB_SERVER_TYPE" = "gunicorn-async" ]; then
+    run_command="WEB_SERVER_WORKER_CLASS=gevent gunicorn application:application"
+elif [ "$WEB_SERVER_TYPE" = "gunicorn-threads" ]; then
+    run_command="WEB_SERVER_WORKER_CLASS=gthread gunicorn application:application"
+elif [ "$WEB_SERVER_TYPE" = "uwsgi" ]; then
     run_command="uwsgi uwsgi.ini --workers ${WEB_SERVER_WORKERS}"
-elif [ "$web_server" = "uwsgi-threads" ]; then
+elif [ "$WEB_SERVER_TYPE" = "uwsgi-threads" ]; then
     run_command="uwsgi uwsgi.ini --workers ${WEB_SERVER_WORKERS} --enable-threads --threads ${WEB_SERVER_THREADS}"
-elif [ "$web_server" = "uwsgi-async" ]; then
-    run_command="uwsgi uwsgi.ini --module patched:application --workers ${WEB_SERVER_WORKERS} --single-interpreter --gevent ${WEB_SERVER_ASYNC_CORES}"
+elif [ "$WEB_SERVER_TYPE" = "uwsgi-async" ]; then
+    run_command="uwsgi uwsgi.ini --module patched:application --workers ${WEB_SERVER_WORKERS} --single-interpreter --gevent ${UWSGI_ASYNC_CORES}"
```

Comment:
I think we need a different environment variable name. uWSGI is parsing this and reading it as a value for an unknown directive `async-cores`:
```
[strict-mode] unknown config directive: async-cores
```

Reply:
Have renamed `UWSGI_ASYNC_CORES` to `WEB_SERVER_UWSGI_ASYNC_CORES`